src/util/log/fd_log.h

```diff
@@ -157,53 +155,40 @@
 #define FD_LOG_ALERT(a)   do { long _fd_log_msg_now = fd_log_wallclock(); fd_log_private_2( 6, _fd_log_msg_now, __FILE__, __LINE__, __func__, fd_log_private_0 a ); } while(0)
 #define FD_LOG_EMERG(a)   do { long _fd_log_msg_now = fd_log_wallclock(); fd_log_private_2( 7, _fd_log_msg_now, __FILE__, __LINE__, __func__, fd_log_private_0 a ); } while(0)
 
-#define FD_LOG_HEXDUMP_BYTES_PER_LINE             16
-#define FD_LOG_HEXDUMP_BLOB_DESCRIPTION_MAX_LEN   32
-#define FD_LOG_HEXDUMP_MAX_INPUT_BLOB_SZ          1500
-
-#define FD_LOG_HEXDUMP_ADD_TO_LOG_BUF(...)  do {                              \
-    num_bytes_written = sprintf ( log_buf_ptr, __VA_ARGS__ );                 \
-    if( FD_LIKELY( num_bytes_written>=0 )) log_buf_ptr += num_bytes_written;  \
- } while(0)
+#define FD_LOG_HEXDUMP_BYTES_PER_LINE             16UL
+#define FD_LOG_HEXDUMP_BLOB_DESCRIPTION_MAX_LEN   32UL
+#define FD_LOG_HEXDUMP_MAX_INPUT_BLOB_SZ          1500UL
```

Comment:
Missed moving these to fd_log_private_hexdump too (and, ultra nit picky, would add parens around the values).

Reply:
Thanks, done. Moved them to be `#define`'d and `#undef`'d within the hexdump msg function.